components/core/src/clp_s/TimestampPattern.hpp

```diff
@@ -43,6 +43,8 @@ namespace clp_s {
  * - S  2-digit 0-padded second (00-60) (60 to account for leap seconds)
  * - 3  0-padded millisecond (000-999)
  * - T  0-padded millisecond no trailing 0 (000)-999) e.g. (000), 9(00), 99(0), 099
+ * - E  N-digit UNIX epoch timestamp in milliseconds
+ * - F  N-digit UNIX epoch timestamp in floating point format with 9-digits trailing the decimal
```

Comment:
```suggestion
 * - E  N-digit millisecond-precision UNIX epoch timestamp
 * - F  N-digit nanosecond-precision UNIX epoch timestamp in floating-point format with 9 digits
 *      trailing the decimal
```

I'm not sure "N-digit" adds much, but I don't mind if you want to leave it in.

Reply:
While we're in the area, does the description for `T` have an extra `)` in it?

Reply:
yep, I can take that out